contracts/TrueUSD.sol

```diff
@@ -65,9 +63,4 @@ GasRefundToken {
         _ownable.transferOwnership(owner);
     }
 
-    function _burnAllArgs(address _burner, uint256 _value) internal {
-        //round down burn amount so that the lowest amount allowed is 1 cent
-        uint burnAmount = _value.div(10 ** uint256(DECIMALS - ROUNDING)).mul(10 ** uint256(DECIMALS - ROUNDING));
```

Comment:
`_value - _value % CENT` is 8 gas cheaper than `(_value / CENT) * CENT`:
```
        ✓ transferFrom (755ms)
 reduceToBurn regression: 89172 -> 89180 
 emptyToBurn regression: 59236 -> 59244 
        ✓ burn (140ms)
 burnMicroDollar regression: 89172 -> 89180 
 reduceToBurnWithChange regression: 89236 -> 89244 
 emptyToBurnWithChange regression: 89236 -> 89244 
        ✓ burn with change (205ms)
      with refund
        ✓ transfer (284ms)
        ✓ transferFrom (817ms)
 reduceToBurnWithRefund regression: 89172 -> 89180 
 emptyToBurnWithRefund regression: 59236 -> 59244 
        ✓ burn (135ms)
 burnMicroDollarWithRefund regression: 89172 -> 89180 
 reduceToBurnWithChangeWithRefund regression: 89236 -> 89244 
 emptyToBurnWithChangeWithRefund regression: 89236 -> 89244 
        ✓ burn with change **(201ms)**
```

Using safemath adds another 204.